### PR TITLE
Travis CI case-study

### PR DESCRIPTION
**Pull request by Ramsnes**
Adds deep-dive into case-study of Travis CI.

### CODE REVIEW

**Review by sulenchy — CHANGES_REQUESTED** [review]
@Ramsnes 
This write-up looks good to me. The presentation and tone are awesome and beginner-friendly. Also, good job on adding an image to boost the level of comprehension of the write-up.

However, I recommend you add a sidebar for the CI of Travis in the `astro-config.mjs`. Additionally, the build is failing, which will prevent me from merging the PR. I suggest you reach out to @jaaneh  regarding this matter.

**Comment by jaaneh** [comment]
The CI fails becuase of the frontmatter. The `title`, `author` and `tags` fields need to be lowercase.

File: src/content/docs/travis-ci/travis-ci-case-study.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
+---
+Title: Travis-CI Case Study
```
Comment (sulenchy) [inline src/content/docs/travis-ci/travis-ci-case-study.md:2]:
```suggestion
title: Travis-CI Case Study
```

File: src/content/docs/travis-ci/travis-ci-case-study.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
+---
+title: Travis-CI Case Study
+Author: Morten Ramfjord, Ramsnes
```
Comment (sulenchy) [inline src/content/docs/travis-ci/travis-ci-case-study.md:3]:
```suggestion
author: Morten Ramfjord, Ramsnes
```

File: src/content/docs/travis-ci/travis-ci-case-study.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
+---
+title: Travis-CI Case Study
+author: Morten Ramfjord, Ramsnes
+Tags: travisci, travis, travis-ci
```
Comment (sulenchy) [inline src/content/docs/travis-ci/travis-ci-case-study.md:4]:
```suggestion
tags: travisci, travis, travis-ci
```

**Comment by sulenchy** [comment]
Fix it the frontmatter. Thanks @jaaneh

**Review by sulenchy — APPROVED** [review]
Looks good now

**Pull request merged**